src/StakingNode.sol

```diff
@@ -194,6 +223,11 @@ contract StakingNode is IStakingNode, StakingNodeEvents, ReentrancyGuardUpgradea
         for (uint256 i = 0; i < validatorIndices.length; i++) {
             uint256 effectiveBalanceGwei = validatorFields[i].getEffectiveBalanceGwei();
             emit ValidatorRestaked(validatorIndices[i], oracleTimestamp, effectiveBalanceGwei);
+
+            // Decrease unverifiedStakedETH by DEFAULT_VALIDATOR_STAKE regardless if the current balance of the validator
+            // Since unverifiedStakedETH was increased by DEFAULT_VALIDATOR_STAKE when the validator was staked
+            // within the Beacon Chain
+            unverifiedStakedETH -= DEFAULT_VALIDATOR_STAKE;
```

Comment:
there's a bug here.

if you decrease this by 32ETH, but verifyWithdrawalCredentials yields for example 0 shares, because validator was withdrawn, balance drops potentially without justification. assets may still be in the system.